src/helpers/helpers.ts

```diff
@@ -62,14 +62,25 @@ export async function copyFileOrDir(
   sourceFileOrDir: string,
   dest: string,
 ): Promise<void> {
-  return new Promise((resolve, reject) => {
-    ncp(sourceFileOrDir, dest, (error: Error[] | null): void => {
-      if (error) {
-        reject(error);
-      }
-      resolve();
-    });
-  });
+  return fs_extra.copy(sourceFileOrDir, dest);
+}
+
+export async function makeFileTreeWriteable(fileOrDir: string): Promise<void> {
+  const mode = 0o755; // read & execute for anyone, plus write for owner
+
+  const stats = await fs_extra.lstat(fileOrDir);
+
+  if (stats.isFile()) {
+    await fs_extra.chmod(fileOrDir, mode);
+  } else if (stats.isDirectory()) {
+    const children = await fs_extra.readdir(fileOrDir, { withFileTypes: true });
+
+    for (const child of children) {
+      await makeFileTreeWriteable(path.resolve(fileOrDir, child.name));
+    }
+
+    await fs_extra.chmod(fileOrDir, mode);
+  }
 }
```

Comment:
Here I used `fs_extra` since a promise is returned when a callback is not given. Also, I didn't want to have to `promisify` all of the calls as I think it would make it less readable